model/otlp/json_test.go

```diff
@@ -55,11 +55,11 @@ var logsOTLP = func() pdata.Logs {
 	il := rl.InstrumentationLibraryLogs().AppendEmpty()
 	il.InstrumentationLibrary().SetName("name")
 	il.InstrumentationLibrary().SetVersion("version")
-	il.LogRecords().AppendEmpty().SetName("testMetric")
+	il.LogRecords().AppendEmpty().SetSeverityText("Error")
```

Comment:
Extremely confused by the choice of replacing this with Severity. Should severity not be an enum?

Reply:
SeverityText is a string (SeverityNumber is an enum). 

In this particular test it does not matter what field we use. This is merely to populate a log record that we later check exists after serialization. The checks are not comprehensive, they only verified a single field (Name) and I did not aim to improve the tests in any way, merely replaced Name by SeverityText to keep the test as is.